src/Common/src/Interop/Unix/libcurl/Interop.libcurl.cs

```diff
@@ -40,5 +65,14 @@ internal static partial class libcurl
             SafeCurlHandle curl,
             int info,
             ref long value);   // Using a ref because it won't be populated on error
+
+        [DllImport(Interop.Libraries.LibCurl, CharSet = CharSet.Ansi)]
+        public static extern IntPtr curl_slist_append(
```

Comment:
The semantics for this are:
     slist = curl_slist_append(slist, "my header")

which suggests that the handle is liable to change. Will this be consistent with SafeHandle semantics?